daemon/status.go

```diff
@@ -32,9 +34,305 @@ import (
 )
 
 const (
+	// collectStatusInterval is how often statuses will be collected
 	collectStatusInterval = 5 * time.Second
+
+	// staleTimeout is how stale a status can be before we report an error
+	staleTimeout = 1 * collectStatusInterval
 )
 
+type getHealthz struct {
+	daemon *Daemon
+}
+
+func NewGetHealthzHandler(d *Daemon) GetHealthzHandler {
+	return &getHealthz{daemon: d}
+}
+
+func (h *getHealthz) Handle(params GetHealthzParams) middleware.Responder {
```

Comment:
Shouldn't this be somewhere else? I had the feeling the "Handles" were all in a single file